basic/src/shared/components/Header.tsx

```diff
@@ -0,0 +1,37 @@
+import { logout, useAuth } from "wasp/client/auth";
+import { Link } from "wasp/client/router";
+import Logo from "../../assets/logo.svg";
+import { Button, ButtonLink } from "./Button";
+
+export function Header() {
+  const { data: user } = useAuth();
+
+  return (
+    <header className="sticky top-0 z-10 flex justify-center border-b border-neutral-200 bg-white text-black shadow">
+      <div className="flex w-full max-w-screen-lg items-center justify-between p-4 px-12">
+        <Link to="/" className="flex items-center gap-2">
+          <img src={Logo} alt="Todo App Logo" className="h-10 w-10" />
+          <h1 className="text-2xl font-semibold">Todo App</h1>
+        </Link>
+        <nav>
+          <ul className="flex gap-x-6 font-semibold">
```

Comment:
```suggestion
          <ul className="flex gap-4 font-semibold">
```

Maybe the gap narrower to make the buttons feel more like group.